django/db/migrations/state.py

```diff
@@ -357,10 +357,9 @@ def from_model(cls, model, exclude_rels=False):
                 continue
             if isinstance(field, OrderWrt):
                 continue
-            name, path, args, kwargs = field.deconstruct()
-            field_class = import_string(path)
+            name = force_text(field.name, strings_only=True)
```

Comment:
Because we need to avoid non ASCII letters in output. It's the same we're doing in https://github.com/django/django/blob/master/django/db/models/fields/__init__.py#L450.